embedding-stores/langchain4j-community-neo4j/src/main/java/dev/langchain4j/community/store/embedding/neo4j/Neo4jEmbeddingStore.java

```diff
@@ -189,6 +195,7 @@ public Neo4jEmbeddingStore(
         this.metadataPrefix = getOrDefault(metadataPrefix, "");
         this.textProperty = getOrDefault(textProperty, DEFAULT_TEXT_PROP);
         this.awaitIndexTimeout = getOrDefault(awaitIndexTimeout, DEFAULT_AWAIT_INDEX_TIMEOUT);
+        this.additionalParams = getOrDefault(additionalParams, Map.of());
```

Comment:
```suggestion
        this.additionalParams = copy(additionalParams);
```

nit: just to keep immutable.